adders/eslint/config/adder.ts

```diff
@@ -0,0 +1,134 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { defineAdderConfig, log } from '@svelte-add/core';
+import { options } from './options.js';
+import { addEslintConfigPrettier } from '../../common.js';
+
+export const adder = defineAdderConfig({
+	metadata: {
+		id: 'eslint',
+		name: 'ESLint',
+		description: 'A configurable JavaScript linter',
+		environments: { svelte: true, kit: true },
+		website: {
+			logo: './eslint.svg',
+			keywords: ['eslint', 'code', 'linter'],
+			documentation: 'https://eslint.org/',
+		},
+	},
+	options,
+	integrationType: 'inline',
+	packages: [
+		{ name: 'eslint', version: '^9.7.0', dev: true },
+		{ name: '@types/eslint', version: '^9.6.0', dev: true },
+		{ name: 'globals', version: '^15.0.0', dev: true },
+		{
+			name: 'typescript-eslint',
+			version: '^8.0.0-alpha.54',
+			dev: true,
+			condition: ({ typescript }) => typescript.installed,
+		},
+		{ name: 'eslint-plugin-svelte', version: '^2.36.0', dev: true },
+		{
+			name: 'eslint-config-prettier',
+			version: '^9.1.0',
+			dev: true,
+			condition: ({ prettier }) => prettier.installed,
+		},
+	],
+	files: [
+		{
+			name: () => 'package.json',
+			contentType: 'json',
+			content: ({ data }) => {
+				data.scripts ??= {};
+				const scripts: Record<string, string> = data.scripts;
+				const LINT_CMD = 'eslint .';
+				scripts['lint'] ??= LINT_CMD;
```

Comment:
Should we also add `eslint --fix .` to the `format` command, as in our repo?

Reply:
we could, but I was basing this off of the existing template that's generated from `create-svelte` with `prettier+eslint` which doesn't include that. I'm leaning towards keeping it as-is and leaving it out

Reply:
Yeah, didn't think about that. I think that aligns with what we were doing with the prettier adder.